src/interpreter/kernel/instruction/unop.js

```diff
@@ -1,6 +1,13 @@
 // @flow
 
-type Sign = "abs" | "neg" | "clz" | "ctz" | "popcnt" | "eqz";
+type Sign =
```

Comment:
It seems that the naming is wrong here

Reply:
Yeah, good point, I've renamed to `Operation`